Quiz Master/src/screens/results/OptionSummary.tsx

```diff
@@ -0,0 +1,30 @@
+import { useSelector } from "react-redux";
+import { types, times, difficulties } from "../../constants/ValueOptions";
+import { RootState } from "../../store";
+
+const OptionSummary = () => {
+  const gameSettings = useSelector((state: RootState) => state.settings);
+  const categories = useSelector((state: RootState) => state.categories.categories);
+  return (
+    <div className="config-summary">
+      <p>Config</p>
+      <p>
+        Type: {types.find((f) => f.value == gameSettings?.type)?.label ?? "Any"}
```

Comment:
While this will work fine, consider extracting those kinda data manipulations to some function. Let's say you can have a `utils` folder, which contains utility functions, one of which could be like the following:

`const getTypeLabel = (currentSettings) => {
  return types.find((f) => f.value == currentSettings?.type)?.label ?? "Any"
}`

It would both increase the readability of your markup and allow you to reuse that method somewhere else